ipyleaflet/leaflet.py

```diff
@@ -139,6 +139,20 @@ class Layer(Widget, InteractMixin):
     pane = Unicode('').tag(sync=True)
 
     options = List(trait=Unicode()).tag(sync=True)
+    subitems = Tuple().tag(trait=Instance(Widget), sync=True, **widget_serialization)
+    _subitem_ids = List()
+
+    @validate('subitems')
+    def _validate_subitems(self, proposal):
+        '''Validate subitems list.
+
+        Makes sure only one instance of any given subitem can exist in the
+        subitem list.
+        '''
+        self._subitem_ids = [subitem.model_id for subitem in proposal.value]
+        if len(set(self._subitem_ids)) != len(self._subitem_ids):
```

Comment:
Do we need `_subitem_ids` to be a class attribute?
```suggestion
        subitem_ids = [subitem.model_id for subitem in proposal.value]
        if len(set(subitem_ids)) != len(subitem_ids):
```

Reply:
It looks like this was marked resolved by mistake? It looks like it's not resolved